### DISCUSSION TITLE
關於彎彎玩家沒有qq那回事

### DISCUSSION THREAD

**Discussion by 123hi123** [Moemu/Muice-Chatbot#45] [Ideas]
想看看能不能用dc bot
我會去研究 如果大家有想法也可以跟我提供讓我少走彎路

**Comment by Moemu** [comment]
今日小知识：其实bot最早的方案真的是dc bot
![image](https://github.com/user-attachments/assets/d4527934-aea1-423a-b917-310985b02940)

**Reply by Moemu** [reply]
不过dc bot的那个框架老是报心跳错误我就没用了

**Comment by Moemu** [comment]
一个额外的问题我想知道一下：请问你们都是通过哪些渠道知道这个项目的？（我不记得我做过宣传）

**Reply by lmx0125** [reply]
on bilibili

**Comment by 123hi123** [comment]
> 今日小知识：其实bot最早的方案真的是dc bot
> 
> ![image](https://github.com/user-attachments/assets/d4527934-aea1-423a-b917-310985b02940)
> 
> 

我能試試嗎
什麼樣的錯誤
我自己有做一個股票祭器人
有一個功能是定時循環查詢
也會報錯
但他還是會繼續他的工作哈哈哈

**Reply by 123hi123** [reply]
還有什麼是基於claude模型 這樣意思是要用要花錢買apiㄇㄚ

**Reply by Moemu** [reply]
以前是不用钱的

**Reply by Moemu** [reply]
都一年以前的了，源码早不见了，不过机器人的账号我还留着

**Reply by 123hi123** [reply]
帳號留著的意思是還能體驗嗎

**Reply by Moemu** [reply]
不能，服务不在，有账号也没地方调用

**Reply by 123hi123** [reply]
好吧
我今天居然能註冊qq了 明天有空就來試試看
不過我顯存只有8 不確定能不能用
我看了一下你的文檔 有提到onebot
這是什麼東西
看起來好像不一定只能部署在qq上的樣子

**Reply by MoeSnowyFox** [reply]
onebot可以在任意有适配器的地方运行哦

**Reply by MoeSnowyFox** [reply]
有许多项目对此有支持
[Gensokyo-Discord](https://github.com/Hoshinonyaruko/Gensokyo-Discord)
[onedisc](https://onedisc.itcdt.top/)等

**Reply by 123hi123** [reply]
多謝 看起來one bot是一個避免重複造輪子的協議

**Comment by 123hi123** [comment]
> 一个额外的问题我想知道一下：请问你们都是通过哪些渠道知道这个项目的？（我不记得我做过宣传）

Ｂ站
